docs/content/integrations/tableau.mdx

```diff
@@ -151,3 +155,34 @@ defs = Definitions.merge(
     marketing_team_workspace.build_defs(),
 )
 ```
+
+### Refresh and materialize Tableau assets
+
+You can use Dagster to orchestrate the refreshment of Tableau workbooks and materialization of Looker sheets and dashboards.
+
+To refresh a Tableau workbook with Dagster, pass its definition by passing `StartWorkbookRefreshRequest` to `TableauCloudWorkspace.build_defs(...)` function.
+
+Note that only workbooks created with extracts can be refreshed using this method. See more about [refreshing data sources](https://help.tableau.com/current/pro/desktop/en-us/refreshing_data.htm) in Tableau documentation website.
+
+When materializing the Tableau assets, Dagster will refresh the workbooks for which a `StartWorkbookRefreshRequest` has been created. Dagster will materialize the sheets and dashboards of workbooks that have been refreshed as `AssetMaterialization`, and the sheets and dashboards of the workbooks that have not been refreshed as `AssetObservation`.
```

Comment:
This is a little confusing. Does a user need to know the distinction here? I think I would remove this whole paragraph. The API docs for `StartWorkbookRefreshRequest` should outline what this function does, but I don't think the guide needs it.

Reply:
That makes sense. All changes requested done in [b954994](https://github.com/dagster-io/dagster/pull/24864/commits/b954994282ac767eee6f80278413b0fe41f43e52)